mobile-app/app/screens/AppNavigator/screens/Settings/components/RowLanguageItem.tsx

```diff
@@ -40,38 +42,45 @@ export function RowLanguageItem ({ languageItem }: { languageItem: AppLanguageIt
     })
   }
 
+  const checkActive = 'bg-success-600'
+  const checkInactive = `${isLight ? 'bg-mono-light-v2-700 bg-opacity-30' : 'bg-mono-dark-v2-700 bg-opacity-30'}`
+
   return (
     <ThemedTouchableOpacity
       onPress={onPress}
-      style={tailwind('flex flex-row p-4 pr-2 items-center justify-between')}
+      style={tailwind('mx-5', { 'rounded-t-lg': firstItem }, { 'rounded-b-lg': lastItem })}
       testID={`button_language_${languageItem.language}`}
+      dark={tailwind('bg-mono-dark-v2-00')}
+      light={tailwind('bg-mono-light-v2-00')}
     >
-      <View>
-        <ThemedText testID='language_option' style={tailwind('font-medium')}>
-          {languageItem.displayName}
-        </ThemedText>
-        <ThemedText
-          testID='language_option_description'
-          dark={tailwind('text-gray-400')}
-          light={tailwind('text-gray-500')}
-          style={tailwind('text-sm')}
+      <View style={tailwind(`flex flex-row mx-5 py-4 items-center justify-between border-b-0.5 ${isLight ? 'border-mono-light-v2-300' : 'border-mono-dark-v2-300'}`, { 'border-b-0': lastItem })}>
```

Comment:
I think we're losing some functionality here. Let's keep it as what we have now on the app.

Display Language (Translated language on the select label).

Reply:
![Screenshot_20220709-112214_DeFiChain Wallet](https://user-images.githubusercontent.com/25013382/178089666-be75ccf1-e4d4-4d91-897a-45eb0d0240f2.jpg)

Reply:
Example, the text below is translated. For the new layout, let's translate also the one inside the ( )